cedar-lean/Cedar/Spec/Request.lean

```diff
@@ -14,6 +14,7 @@
  limitations under the License.
 -/
 
+import Cedar.Data.Map
```

Comment:
Nit: why the new import here?

Reply:
This file directly/explicitly uses `Map`, and I find it better style to explicitly import things that we explicitly use, rather than relying on Lean's transitive imports